views/templates/pages/index.tpl

```diff
@@ -0,0 +1,14 @@
+<div class="container <?= get_data('cls'); ?>"<?php
+    foreach(get_data('data') as $name => $value) {
+        echo ' data-' . $name . '="' .(is_string($value) ? $value : _dh(json_encode($value))) . '"';
```

Comment:
No I meant the formatting of the value itself  `(is_string($value) ? $value : _dh(json_encode($value)))` shouldn't be made in the template.